files/SSRF.txt

```diff
@@ -0,0 +1 @@
+Your attack was successful!!
```

Comment:
we don't need this, instead create a github gist page (I will create for you which has all the project names by SasanLabs)